consensus/spos/bls/v2/subroundBlock.go

```diff
@@ -455,22 +375,23 @@ func (sr *subroundBlock) addProofOnHeader(header data.HeaderHandler) bool {
 	// in this case, we set the previous proof as signature and bitmap from the previous header
 	// 2. current node is leader in the first block after sync
 	// in this case, we won't set the proof, return false and wait for the next round to receive a proof
-	currentHeader := sr.Blockchain().GetCurrentBlockHeader()
-	if check.IfNil(currentHeader) {
+	prevBlockHeader := sr.Blockchain().GetCurrentBlockHeader()
+	if check.IfNil(prevBlockHeader) {
 		log.Debug("addProofOnHeader.GetCurrentBlockHeader, nil current header")
```

Comment:
update maybe also log messages here with prev block header?

Reply:
I will take this on the next PR